dspace/config/spring/api/external-services.xml

```diff
@@ -92,5 +94,26 @@
         </property>
     </bean>
 
-</beans>
+    <bean id="crossRefLiveImportDataProvider" class="org.dspace.external.provider.impl.LiveImportDataProvider">
+        <property name="metadataSource" ref="CrossRefImportService"/>
+        <property name="sourceIdentifier" value="crossref"/>
+        <property name="recordIdMetadata" value="dc.identifier.doi"/>
+        <property name="supportedEntityTypes">
+            <list>
+                <value>Publication</value>
```

Comment:
Can these data providers also be used for regular Items?  or do they require Publication entities?

Reply:
My question here was never answered about whether these import tools only work for `Publication` or if they also can support `none` (i.e. regular Items).  That said, we can leave this as-is for now if we don't feel comfortable supporting regular Items.

Reply:
@tdonohue sorry for delay, yes it also be used for regular Items. I have already applied the changes.